unique_toolkit/unique_toolkit/chat/schemas.py

```diff
@@ -42,18 +42,18 @@ class ChatMessage(BaseModel):
     model_config = model_config
 
     id: str | None = None
-    chat_id: str
+    chat_id: str | None = None
     object: str | None = None
     content: str = Field(alias="text")
-    original_content: str | None = Field(alias="originalText")
+    original_content: str | None = None
```

Comment:
The alias here is very important. Cannot remove this. Please check if it is optional to determine the exact type from our API